### ISSUE TITLE
[Spell Checker] Multiple dictionaries feature and some weirdness

### ISSUE THREAD

**Issue by spaeps** [QupZilla/qupzilla#844]
I'd like to let hunspell check two or more languages at the same time,
as midori does too (ex. you can use both "en" and "it", it just sums up
the content of the dictionaries).

Also, I notice spell checking works this way in qupzilla:

wrong words are underlined on the second (or sometimes third)
character typed after the spacing.

Ex, writing the sentence "This is wron you know",
the world "wron" is underlined (in red) only once the "o" letter in "you" is typed.

This is not as it should work.
The wrong word should be underlined as the first space character is typed.

Also, selecting another dictionary in a text area, does not refresh the spell checking on the already written text, and you have to click on every word in order to be checked.
This is also wrong or noisy: text should be rechecked on dictionary change.

So, to sum up:
1. multiple dictionary feature
2. check the word as the first space character after the word is typed
3. refresh spell checking on a text, when a different dictionary is selected

thanks
speps

**Comment by nowrep** [comment]
Multiple dictionaries are now supported with QtWebEngine.
